src/databricks/labs/ucx/assessment/jobs.py

```diff
@@ -140,6 +145,16 @@ def _check_jar_task(self, all_task: list[RunTask]) -> list[str]:
         return task_failures
 
 
+class JobOwnership(Ownership[JobInfo]):
+    """Determine ownership of jobs (workflows) in the inventory.
+
+    This is the pipeline creator (if known), or otherwise an administrator.
+    """
+
+    def _get_owner(self, record: JobInfo) -> str | None:
+        return record.creator
```

Comment:
what about `run_as`?

Reply:
This is interesting. This is not being gathered by the crawler, which means either: a) updating the crawler to include it; b) doing it here.

Wherever this is done, it looks more complicated than at first glance: the `run_as` property is documented on the SDK as write-only. I'll do some tests to see if this is really the case or not, but if that's correct then it needs to be found somewhere else and I'm not (yet) sure where?

Either way I'd suggest covering this as a separate PR.